roles/BCPC-Hadoop-Head-HBase.json

```diff
@@ -4,6 +4,7 @@
   "run_list": [
     "role[Basic]",
     "role[BCPC-Hadoop-Head]",
+    "recipe[bcpc-hadoop::hbase_repl]",
```

Comment:
The idea is that ``chef-bach`` can be used to create two clusters and users can implement ``hbase`` replication. As acknowledged since the default value is set to ``false`` for the attribute to enable replication, it should not impact any user who is creating a single cluster using ``chef-bach``